app/components/eventSearchBar/SearchBar.tsx

```diff
@@ -0,0 +1,20 @@
+
+
+export default function SearchBar() {
+    return (
+        <>
```

Comment:
why is fragment tag needed here ?

Reply:
To avoid errors, either create a div tag or use this fragment to bind your code.

Reply:
if you have  more than one view inside the return then you need empty fragment or some view to wrap childrens, otherwise no need of this